shipment_advice/data/queue_job_channel.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+  <record model="queue.job.channel" id="shipment_advice_queue_channel">
+    <field name="name">background.shipment_advice</field>
```

Comment:
@sbejaoui The name should not contains '.' `background.` refers to the name of the parent queue...